src/App.jsx

```diff
@@ -1,107 +1,31 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
+}
+
+function getCommentsByPostId(postId) {
+  return commentsFromServer.filter(
+    comment => comment.postId === postId,
+  )
+      || null;
```

Comment:
Do we need to return null if we check the comments array's length in the CommentList component?